__tests__/app.test.js

```diff
@@ -172,43 +172,122 @@ describe('POST /api/articles/:article_id/comments/', () => {
       .post("/api/articles/2/comments/")
       .send({
         body: "Laptops are useful!",
-        author: "rogersop",
-        article_id: 2
-      })
-      .expect(200)
+        author: "rogersop"      })
+      .expect(201)
       .then(({ body}) => {
-        expect(body).toMatchObject({
+        expect(body.comment).toMatchObject({
           comment_id: 19,
           body: 'Laptops are useful!',
           article_id: 2,
           author: 'rogersop'
         })
       })
   });
-  test("404:error when an incorrect article ID is entered e.g.'123456789' ", () => {
+  test("404: Not found error when an incorrect article ID is entered e.g.'123456789' ", () => {
     return request(app)
       .post("/api/articles/123456789/comments")
       .send({
         body: "I like pears!",
         author: "rogersop",
-        article_id: 123456789
       })
       .expect(404)
       .then(({ body }) => {
         expect(body.msg).toBe("Not found");
       });
   });
 
-  test('should return a bad request error if no comment is sent ', () => {
+  test('400: Bad request error if no comment is sent ', () => {
     return request(app)
       .post("/api/articles/2/comments")
       .send({
-        author: "rogersop",
-        article_id: 2
+        body: "",
+        author: "rogersop"
       })
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toBe("Bad request");
       });
   });
+
+  test('400: Bad request error if invalid article_id', () => {
+    return request(app)
+      .post("/api/articles/bananas/comments")
+      .send({
+        body: "I like bananas!",
+        author: "rogersop"
+      })
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+
+  test('404: Not found error if author does not exist', () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send({
+        body: "I like apples!",
+        author: "sandra",
+      })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Not found");
+      });
+  });
+});
+
+describe('PATCH /api/articles/:article_id/', () => {
+  test('when the article votes property is missing, it should add votes to an article with the matching article_id', () => {
+    return request(app)
+    .patch('/api/articles/3/')
+    .send({inc_votes : 10 })
+    .then(({ body: { article } }) => {
+      expect(article).toMatchObject({
+        article_id: 3,
+        votes: 10
+      })
+  });
```

Comment:
You might want to check all the properties of the article object are correctly returned to make this test more robust 💡